### PR TITLE
orbitron: v2.000 added.

### PR DESCRIPTION
**Pull request by eliheuer**
Taken from the upstream repo https://github.com/eliheuer/orbitron-vf
at commit https://github.com/eliheuer/orbitron-vf/commit/6ab11dd90075bf7d6604e6c9008b51337766928f

This is a new PR from `google/fonts:orbitron` that replaces an old PR from `eliheuer/fonts:orbitron`.
The old PR will be closed soon and can be found here: https://github.com/google/fonts/pull/1743

### CODE REVIEW

**Comment by m4rc1e** [comment]
@felipesanches I'm not sure if this glyph coverage check is overzealous for existing fonts. @eliheuer has matched the previous glyphset, minus one pua which is acceptable. In this instance, I'm going to ignore this check.

File: ofl/orbitron/OFL.txt [hunk 1]
```diff
@@ -1,94 +1,95 @@
...
-MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
-OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE
-COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
-INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
-DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
-OTHER DEALINGS IN THE FONT SOFTWARE.
+Copyright 2018 The Orbitron Project Authors (https://github.com/theleagueof/orbitron).
+with Reserved Font Name: "Orbitron".
+(https://github.com/theleagueof/orbitron)
```
Comment (m4rc1e) [inline ofl/orbitron/OFL.txt]: Drop this url

Comment (eliheuer) [reply ofl/orbitron/OFL.txt]: Ok, I just pushed an updated that I think fixes all there issues

**Comment by eliheuer** [comment]
Ok, I just pushed an updated that should resolve all the above issues.

**Comment by eliheuer** [comment]
Is there anything that is keeping this from being merged? I'm happy to make updates I just don't see what needs to be done at this point. Thanks Marc!

**Comment by m4rc1e** [comment]
We're good. I'm going to start pushing next week.

**Comment by m4rc1e** [comment]
@eliheuer actually, update the DESCRIPTION.en_us.html so it mentions the upstream repo url and the font is now a VF.

**Comment by eliheuer** [comment]
Ah yes, good idea. The `The DESCRIPTION.en_us.html` file has been updated. I need to pr my changes back to the upstream at some point, but the quadratic drawing and fontmake build process complicates that a little.

**Comment by eliheuer** [comment]
I looked over this today and it should be ready to merge, let me know if it's still missing anything @m4rc1e . Thanks!

**Comment by eliheuer** [comment]
I also updated this PR to use the new VF naming spec as was discussed here: https://github.com/google/fonts/issues/1817

**Comment by m4rc1e** [comment]
New VF filename `Orbitron\[wght\].ttf` causes fontbakery to skip. This issue has been reported so I won't bring it up.

**Pull request merged**